src/wp-includes/blocks/categories.php

```diff
@@ -81,10 +80,9 @@ function onCatChange() {
 		}
 		dropdown.onchange = onCatChange;
 	})();
-	/* ]]> */
 	</script>
 	<?php
-	return ob_get_clean();
+	return wp_get_inline_script_tag( str_replace( array( '<script>', '</script>' ), '', ob_get_clean() ) );
```

Comment:
This needs to rather be updated in Gutenberg:

https://github.com/WordPress/gutenberg/blob/23bb9301902bb5825480240bd5aaec9d3cc3c380/packages/block-library/src/categories/index.php#L70-L88

Reply:
If you could open a quick Gutenberg PR, that would be great. Then maybe we can still get it merged to core for 6.4. Should be a trivial change in any case :)

Reply:
Done! https://github.com/WordPress/gutenberg/pull/54637

Reply:
Could you remove this from here now so that it doesn't "fail" tests?